README.md

```diff
@@ -17,14 +17,17 @@ designed as libraries (for programmatic use) with minimal command line interface
 
 ## Installation
 
-After cloning the Git repository, on the command line do execute
+To get setup with the OSS Review Toolkit, simply:
```

Comment:
Granted, you're the the native speaker, but shouldn't "setup" be "set up" here? Or maybe even better, simply "started"?